### PR TITLE
refactor: simplify page container

### PR DESCRIPTION
**Pull request by eric-zero-dev**
[Associated Notion Card](https://www.notion.so/dcf93eee667d4edeac4b2a927d211551?v=0fbbeae3210b49af8266e3495e7d8ba5&p=af6e662b7ae442e299e1c74138fb4f63)

-----

## 1. Pull request checklist

- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

### CODE REVIEW

**Review by domw30 — APPROVED** [review]
looks good, nice one 👍

File: src/containers/PageContainer/index.tsx [hunk 1]
```diff
@@ -1,239 +1,108 @@
...
+				localStorage.getItem(LOCAL_STORAGE_KEYS.CHOOSEN_WALLET) as WALLETS,
+			) &&
 			!active &&
 			triedEagerConnect
 		) {
-			setTimeout(async () => {
-				localStorage.removeItem(LOCAL_STORAGE_KEYS.CHOOSEN_WALLET);
-			}, 2000);
+			localStorage.removeItem(LOCAL_STORAGE_KEYS.CHOOSEN_WALLET);
 		}
```
Comment (colbr) [inline src/containers/PageContainer/index.tsx:69]: Do we gain much from using `useCallback` here vs. just having inline functions? I have a feeling that `useCallback` might be less efficient here than inline functions, but might be worth running through Profiler.

File: src/containers/PageContainer/index.tsx [hunk 2]
```diff
@@ -1,239 +1,108 @@
...
-				style={{
-					opacity: hasLoaded ? 1 : 0,
-					transition: 'opacity 0.2s ease-in-out',
-				}}
+				className={classnames(styles.PageContainer, {
+					[styles.isVisible]:
+						Boolean(znsDomain) ||
+						Boolean(prevZnsDomain) ||
+						Boolean(globalDomain.domain),
+				})}
```
Comment (colbr) [inline src/containers/PageContainer/index.tsx:100]: FYI you can use `classnames/bind` for this!

```js
import classnames from 'classnames/bind'
const cx = classnames.bind(styles)

// in element classNames prop
cx(styles.PageContainer, {
    isVisible: something that evaluates to a boolean
})
```

The above will conditionally apply the `ModuleClassName` class

**Review by colbr — APPROVED** [review]
Nice - looks good. Couple nits here and there, but overall v nice.

Will hold off on merging until zAuction v2 release is done - I'll handle any conflicts that crop up from now until then 😊

**Pull request merged**